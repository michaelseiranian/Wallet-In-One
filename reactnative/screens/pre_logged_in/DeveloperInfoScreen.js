import {
    Linking,
    StyleSheet,
    Text,
    TouchableOpacity,
    View,
    Image,
    ScrollView,
    ImageBackground,
    div,
  } from "react-native";
import React from "react";
import { useTheme } from 'reactnative/src/theme/ThemeProvider';

export default function DeveloperInfoScreen ({ navigation }) {

    const {dark, colors, setScheme} = useTheme();

    const styles = StyleSheet.create({
        aboutUs: {
            backgroundColor: colors.primary,
            color: 'black',
            width: "60%",
            borderRadius: 25,
            textAlign: 'center',
            fontWeight: 'bold',
            padding: "2%",
            fontSize:  20,
            alignSelf: 'center',
        },
        developerName: {
            fontSize: 18,
            color: "#fff",
            textTransform: "uppercase",
            fontWeight: "500",
            marginLeft: '-57%',
            alignSelf: "center",
        },
        developerPage: {
            marginTop: 20,
        },
        background: {
            width: '100%',
            height: '100%'
        },
        iconStyle: {
            width: "100%",
            height: 50,
            aspectRatio: 1,
        },
        buttonStyle: {
            borderRadius: 5,
            paddingVertical: -10,
            paddingHorizontal: 28,
            marginRight: '60%',
            display: "flex",
            justifyContent: "center",
            alignItems: "center",
            marginVertical: 15,
        },
        meetTheTeam: {
            fontSize: 18,
            color: "#fff",
            textTransform: "uppercase",
            fontWeight: "500",
            marginVertical: 15,
            alignSelf: "center",
        },
        developersPara: {
            color: "#fff",
        },
        paraStyle: {
            fontSize: 16,
            color: colors.primary,
            paddingBottom: 25,
        },
        textBoxLayout: {
            backgroundColor: colors.primary,
            paddingHorizontal: 30,
            marginVertical: 30,
        },
        logo:{
            width: 290,
            height: 280,
            marginTop: '-10%',
            marginBottom: '-15%',
            alignSelf:'center'
        },
    })

    return (
        <ImageBackground
            source={require('reactnative/assets/background.jpg')}
            style={styles.background}
        >
        <ScrollView
        contentContainerStyle={{
            flexGrow : 1,
            justifyContent : 'center',
            paddingBottom: 15
            }}
        style={styles.developerPage}
        >

        <Image
            source={require('reactnative/assets/logo.png')}
            style={styles.logo}
            resizeMode="contain"
        >
        </Image>

        <View
        contentContainerStyle={{
            paddingBottom: 30,
            }}
        style={styles.textBoxLayout}
        >

        <Text style={styles.meetTheTeam}> Meet the team </Text>
        <Text style={[styles.paraStyle, styles.developersPara]}>
            This app was developed by second year students from King's College 
            London on the Computer Science course for the module 'Software
            Engineering Group Project'. {"\n"}
            Our team totals 8 students, feel free to visit our Github accounts
            and view our other projects by tapping on the icons below.
        </Text>
        </View>

            <View style={{ flexDirection:"row"}}>    
                <TouchableOpacity
                style={styles.buttonStyle}
                onPress={() =>
                    Linking.openURL(
                    "https://github.com/AbbasBinVakas"
                    )
                }
                testID={'GithubButtonTestAbbas'}>
                <Image
                style={styles.iconStyle}
                source={require('reactnative/assets/github.png')}
                />
                </TouchableOpacity>
                <Text style={styles.developerName}> Abbas Bin Vakas </Text>
            </View>
            <View style={{ flexDirection:"row"}}>    
                <TouchableOpacity
                style={styles.buttonStyle}
                onPress={() =>
                    Linking.openURL(
                    "https://github.com/YusufKCL"
                    )
                }
                testID={'GithubButtonTestYusuf'}>
                <Image
                style={styles.iconStyle}
                source={require('reactnative/assets/github.png')}
                />
                </TouchableOpacity>
                <Text style={styles.developerName}> Yusuf Abdulrahman </Text>
            </View>
            <View style={{ flexDirection:"row"}}>    
                <TouchableOpacity
                style={styles.buttonStyle}
                onPress={() =>
                    Linking.openURL(
                    "https://github.com/XEZ1"
                    )
                }
                testID={'GithubButtonTestEzat'}>
                <Image
                style={styles.iconStyle}
                source={require('reactnative/assets/github.png')}
                />
                </TouchableOpacity>
                <Text style={styles.developerName}> Ezzat Alsalibi </Text>
            </View>
            <View style={{ flexDirection:"row"}}>    
                <TouchableOpacity
                style={styles.buttonStyle}
                onPress={() =>
                    Linking.openURL(
                    "https://github.com/Shozab-N18"
                    )
                }
                testID={'GithubButtonTestShozab'}>
                <Image
                style={styles.iconStyle}
                source={require('reactnative/assets/github.png')}
                />
                </TouchableOpacity>
                <Text style={styles.developerName}> Shozab Anwar Siddique </Text>
            </View>
            <View style={{ flexDirection:"row"}}>    
                <TouchableOpacity
                style={styles.buttonStyle}
                onPress={() =>
                    Linking.openURL(
                    "https://github.com/JamalBoustani"
                    )
                }
                testID={'GithubButtonTestJamal'}>
                <Image
                style={styles.iconStyle}
                source={require('reactnative/assets/github.png')}
                />
                </TouchableOpacity>
                <Text style={styles.developerName}> Jamal Boustani </Text>
            </View>
            <View style={{ flexDirection:"row"}}>    
                <TouchableOpacity
                style={styles.buttonStyle}
                onPress={() =>
                    Linking.openURL(
                    "https://github.com/krishnapk7"
                    )
                }
                testID={'GithubButtonTestKrishna'}>
                <Image
                style={styles.iconStyle}
                source={require('reactnative/assets/github.png')}
                />
                </TouchableOpacity>
                <Text style={styles.developerName}> Krishna Prasanna Kumar </Text>
            </View>
            <View style={{ flexDirection:"row"}}>    
                <TouchableOpacity
                style={styles.buttonStyle}
                onPress={() =>
                    Linking.openURL(
                    "https://github.com/mohawk49"
                    )
                }
                testID={'GithubButtonTestMichael'}>
                <Image
                style={styles.iconStyle}
                source={require('reactnative/assets/github.png')}
                />
                </TouchableOpacity>
                <Text style={styles.developerName}> Michael Seiranian </Text>
            </View>
            <View style={{ flexDirection:"row"}}>    
                <TouchableOpacity
                style={styles.buttonStyle}
                onPress={() =>
                    Linking.openURL(
                    "https://github.com/mrmatyog"
                    )
                }
                testID={'GithubButtonTestMatushan'}>
                <Image
                style={styles.iconStyle}
                source={require('reactnative/assets/github.png')}
                />
                </TouchableOpacity>
                <Text style={styles.developerName}> Matushan Yogaraj </Text>
            </View>

            <TouchableOpacity
                onPress={() => navigation.navigate('About Us')}
            >
            <Text style={styles.aboutUs}>Back to About Us</Text>
            </TouchableOpacity>  
        </ScrollView>   
        </ImageBackground>
    )
}