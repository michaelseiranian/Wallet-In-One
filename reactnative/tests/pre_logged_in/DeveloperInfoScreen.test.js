import React from 'react';
import { render, fireEvent } from '@testing-library/react-native';
import DeveloperInfoScreen from '../../screens/pre_logged_in/DeveloperInfoScreen';
import { Linking } from 'react-native';
import renderer from 'react-test-renderer';

describe('<DeveloperInfoScreen />', () => {
    it('back to about us button test', () => {
        const navigate = jest.fn();
        const { getByText } = render(<DeveloperInfoScreen  navigation={{ navigate }}/>);
        fireEvent.press(getByText('Back to About Us'));
        expect(navigate).toHaveBeenCalledWith('About Us');
    })

    it('github button test krishna', () => {
        const github = jest.spyOn(Linking, 'openURL');
        const { getByTestId } = render(<DeveloperInfoScreen/>);
        fireEvent.press(getByTestId('GithubButtonTestKrishna'));
        expect(github).toHaveBeenCalledWith('https://github.com/krishnapk7');
    })

    it('github button test abbas', () => {
        const github = jest.spyOn(Linking, 'openURL');
        const { getByTestId } = render(<DeveloperInfoScreen/>);
        fireEvent.press(getByTestId('GithubButtonTestAbbas'));
        expect(github).toHaveBeenCalledWith('https://github.com/AbbasBinVakas');
    })

    it('github button test yusuf', () => {
        const github = jest.spyOn(Linking, 'openURL');
        const { getByTestId } = render(<DeveloperInfoScreen/>);
        fireEvent.press(getByTestId('GithubButtonTestYusuf'));
        expect(github).toHaveBeenCalledWith('https://github.com/YusufKCL');
    })

    it('github button test ezat', () => {
        const github = jest.spyOn(Linking, 'openURL');
        const { getByTestId } = render(<DeveloperInfoScreen/>);
        fireEvent.press(getByTestId('GithubButtonTestEzat'));
        expect(github).toHaveBeenCalledWith('https://github.com/XEZ1');
    })

    it('github button test shozab', () => {
        const github = jest.spyOn(Linking, 'openURL');
        const { getByTestId } = render(<DeveloperInfoScreen/>);
        fireEvent.press(getByTestId('GithubButtonTestShozab'));
        expect(github).toHaveBeenCalledWith('https://github.com/Shozab-N18');
    })

    it('github button test jamal', () => {
        const github = jest.spyOn(Linking, 'openURL');
        const { getByTestId } = render(<DeveloperInfoScreen/>);
        fireEvent.press(getByTestId('GithubButtonTestJamal'));
        expect(github).toHaveBeenCalledWith('https://github.com/JamalBoustani');
    })

    it('github button test michael', () => {
        const github = jest.spyOn(Linking, 'openURL');
        const { getByTestId } = render(<DeveloperInfoScreen/>);
        fireEvent.press(getByTestId('GithubButtonTestMichael'));
        expect(github).toHaveBeenCalledWith('https://github.com/mohawk49');
    })

    it('github button test matushan', () => {
        const github = jest.spyOn(Linking, 'openURL');
        const { getByTestId } = render(<DeveloperInfoScreen/>);
        fireEvent.press(getByTestId('GithubButtonTestMatushan'));
        expect(github).toHaveBeenCalledWith('https://github.com/mrmatyog');
    })

    it('snapshot developerInfoScreen test', () => {
        const developerInfoScreen = renderer.create(<DeveloperInfoScreen />);
        expect(developerInfoScreen).toMatchSnapshot();         
      });
})