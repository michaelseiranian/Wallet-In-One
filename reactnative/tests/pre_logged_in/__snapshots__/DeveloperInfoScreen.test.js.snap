// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DeveloperInfoScreen /> snapshot developerInfoScreen test 1`] = `
<View
  accessibilityIgnoresInvertColors={true}
  style={
    {
      "height": "100%",
      "width": "100%",
    }
  }
>
  <Image
    source={1}
    style={
      [
        {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        },
        {
          "height": "100%",
          "width": "100%",
        },
        undefined,
      ]
    }
  />
  <RCTScrollView
    contentContainerStyle={
      {
        "flexGrow": 1,
        "justifyContent": "center",
        "paddingBottom": 15,
      }
    }
    style={
      {
        "marginTop": 20,
      }
    }
  >
    <View>
      <Image
        resizeMode="contain"
        source={1}
        style={
          {
            "alignSelf": "center",
            "height": 280,
            "marginBottom": "-15%",
            "marginTop": "-10%",
            "width": 290,
          }
        }
      />
      <View
        contentContainerStyle={
          {
            "paddingBottom": 30,
          }
        }
        style={
          {
            "backgroundColor": "purple",
            "marginVertical": 30,
            "paddingHorizontal": 30,
          }
        }
      >
        <Text
          style={
            {
              "alignSelf": "center",
              "color": "#fff",
              "fontSize": 18,
              "fontWeight": "500",
              "marginVertical": 15,
              "textTransform": "uppercase",
            }
          }
        >
           Meet the team 
        </Text>
        <Text
          style={
            [
              {
                "color": "purple",
                "fontSize": 16,
                "paddingBottom": 25,
              },
              {
                "color": "#fff",
              },
            ]
          }
        >
          This app was developed by second year students from King's College London on the Computer Science course for the module 'Software Engineering Group Project'. 
          

          Our team totals 8 students, feel free to visit our Github accounts and view our other projects by tapping on the icons below.
        </Text>
      </View>
      <View
        style={
          {
            "flexDirection": "row",
          }
        }
      >
        <View
          accessibilityState={
            {
              "busy": undefined,
              "checked": undefined,
              "disabled": undefined,
              "expanded": undefined,
              "selected": undefined,
            }
          }
          accessibilityValue={
            {
              "max": undefined,
              "min": undefined,
              "now": undefined,
              "text": undefined,
            }
          }
          accessible={true}
          collapsable={false}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            {
              "alignItems": "center",
              "borderRadius": 5,
              "display": "flex",
              "justifyContent": "center",
              "marginRight": "60%",
              "marginVertical": 15,
              "opacity": 1,
              "paddingHorizontal": 28,
              "paddingVertical": -10,
            }
          }
          testID="GithubButtonTestAbbas"
        >
          <Image
            source={1}
            style={
              {
                "aspectRatio": 1,
                "height": 50,
                "width": "100%",
              }
            }
          />
        </View>
        <Text
          style={
            {
              "alignSelf": "center",
              "color": "#fff",
              "fontSize": 18,
              "fontWeight": "500",
              "marginLeft": "-57%",
              "textTransform": "uppercase",
            }
          }
        >
           Abbas Bin Vakas 
        </Text>
      </View>
      <View
        style={
          {
            "flexDirection": "row",
          }
        }
      >
        <View
          accessibilityState={
            {
              "busy": undefined,
              "checked": undefined,
              "disabled": undefined,
              "expanded": undefined,
              "selected": undefined,
            }
          }
          accessibilityValue={
            {
              "max": undefined,
              "min": undefined,
              "now": undefined,
              "text": undefined,
            }
          }
          accessible={true}
          collapsable={false}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            {
              "alignItems": "center",
              "borderRadius": 5,
              "display": "flex",
              "justifyContent": "center",
              "marginRight": "60%",
              "marginVertical": 15,
              "opacity": 1,
              "paddingHorizontal": 28,
              "paddingVertical": -10,
            }
          }
          testID="GithubButtonTestYusuf"
        >
          <Image
            source={1}
            style={
              {
                "aspectRatio": 1,
                "height": 50,
                "width": "100%",
              }
            }
          />
        </View>
        <Text
          style={
            {
              "alignSelf": "center",
              "color": "#fff",
              "fontSize": 18,
              "fontWeight": "500",
              "marginLeft": "-57%",
              "textTransform": "uppercase",
            }
          }
        >
           Yusuf Abdulrahman 
        </Text>
      </View>
      <View
        style={
          {
            "flexDirection": "row",
          }
        }
      >
        <View
          accessibilityState={
            {
              "busy": undefined,
              "checked": undefined,
              "disabled": undefined,
              "expanded": undefined,
              "selected": undefined,
            }
          }
          accessibilityValue={
            {
              "max": undefined,
              "min": undefined,
              "now": undefined,
              "text": undefined,
            }
          }
          accessible={true}
          collapsable={false}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            {
              "alignItems": "center",
              "borderRadius": 5,
              "display": "flex",
              "justifyContent": "center",
              "marginRight": "60%",
              "marginVertical": 15,
              "opacity": 1,
              "paddingHorizontal": 28,
              "paddingVertical": -10,
            }
          }
          testID="GithubButtonTestEzat"
        >
          <Image
            source={1}
            style={
              {
                "aspectRatio": 1,
                "height": 50,
                "width": "100%",
              }
            }
          />
        </View>
        <Text
          style={
            {
              "alignSelf": "center",
              "color": "#fff",
              "fontSize": 18,
              "fontWeight": "500",
              "marginLeft": "-57%",
              "textTransform": "uppercase",
            }
          }
        >
           Ezzat Alsalibi 
        </Text>
      </View>
      <View
        style={
          {
            "flexDirection": "row",
          }
        }
      >
        <View
          accessibilityState={
            {
              "busy": undefined,
              "checked": undefined,
              "disabled": undefined,
              "expanded": undefined,
              "selected": undefined,
            }
          }
          accessibilityValue={
            {
              "max": undefined,
              "min": undefined,
              "now": undefined,
              "text": undefined,
            }
          }
          accessible={true}
          collapsable={false}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            {
              "alignItems": "center",
              "borderRadius": 5,
              "display": "flex",
              "justifyContent": "center",
              "marginRight": "60%",
              "marginVertical": 15,
              "opacity": 1,
              "paddingHorizontal": 28,
              "paddingVertical": -10,
            }
          }
          testID="GithubButtonTestShozab"
        >
          <Image
            source={1}
            style={
              {
                "aspectRatio": 1,
                "height": 50,
                "width": "100%",
              }
            }
          />
        </View>
        <Text
          style={
            {
              "alignSelf": "center",
              "color": "#fff",
              "fontSize": 18,
              "fontWeight": "500",
              "marginLeft": "-57%",
              "textTransform": "uppercase",
            }
          }
        >
           Shozab Anwar Siddique 
        </Text>
      </View>
      <View
        style={
          {
            "flexDirection": "row",
          }
        }
      >
        <View
          accessibilityState={
            {
              "busy": undefined,
              "checked": undefined,
              "disabled": undefined,
              "expanded": undefined,
              "selected": undefined,
            }
          }
          accessibilityValue={
            {
              "max": undefined,
              "min": undefined,
              "now": undefined,
              "text": undefined,
            }
          }
          accessible={true}
          collapsable={false}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            {
              "alignItems": "center",
              "borderRadius": 5,
              "display": "flex",
              "justifyContent": "center",
              "marginRight": "60%",
              "marginVertical": 15,
              "opacity": 1,
              "paddingHorizontal": 28,
              "paddingVertical": -10,
            }
          }
          testID="GithubButtonTestJamal"
        >
          <Image
            source={1}
            style={
              {
                "aspectRatio": 1,
                "height": 50,
                "width": "100%",
              }
            }
          />
        </View>
        <Text
          style={
            {
              "alignSelf": "center",
              "color": "#fff",
              "fontSize": 18,
              "fontWeight": "500",
              "marginLeft": "-57%",
              "textTransform": "uppercase",
            }
          }
        >
           Jamal Boustani 
        </Text>
      </View>
      <View
        style={
          {
            "flexDirection": "row",
          }
        }
      >
        <View
          accessibilityState={
            {
              "busy": undefined,
              "checked": undefined,
              "disabled": undefined,
              "expanded": undefined,
              "selected": undefined,
            }
          }
          accessibilityValue={
            {
              "max": undefined,
              "min": undefined,
              "now": undefined,
              "text": undefined,
            }
          }
          accessible={true}
          collapsable={false}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            {
              "alignItems": "center",
              "borderRadius": 5,
              "display": "flex",
              "justifyContent": "center",
              "marginRight": "60%",
              "marginVertical": 15,
              "opacity": 1,
              "paddingHorizontal": 28,
              "paddingVertical": -10,
            }
          }
          testID="GithubButtonTestKrishna"
        >
          <Image
            source={1}
            style={
              {
                "aspectRatio": 1,
                "height": 50,
                "width": "100%",
              }
            }
          />
        </View>
        <Text
          style={
            {
              "alignSelf": "center",
              "color": "#fff",
              "fontSize": 18,
              "fontWeight": "500",
              "marginLeft": "-57%",
              "textTransform": "uppercase",
            }
          }
        >
           Krishna Prasanna Kumar 
        </Text>
      </View>
      <View
        style={
          {
            "flexDirection": "row",
          }
        }
      >
        <View
          accessibilityState={
            {
              "busy": undefined,
              "checked": undefined,
              "disabled": undefined,
              "expanded": undefined,
              "selected": undefined,
            }
          }
          accessibilityValue={
            {
              "max": undefined,
              "min": undefined,
              "now": undefined,
              "text": undefined,
            }
          }
          accessible={true}
          collapsable={false}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            {
              "alignItems": "center",
              "borderRadius": 5,
              "display": "flex",
              "justifyContent": "center",
              "marginRight": "60%",
              "marginVertical": 15,
              "opacity": 1,
              "paddingHorizontal": 28,
              "paddingVertical": -10,
            }
          }
          testID="GithubButtonTestMichael"
        >
          <Image
            source={1}
            style={
              {
                "aspectRatio": 1,
                "height": 50,
                "width": "100%",
              }
            }
          />
        </View>
        <Text
          style={
            {
              "alignSelf": "center",
              "color": "#fff",
              "fontSize": 18,
              "fontWeight": "500",
              "marginLeft": "-57%",
              "textTransform": "uppercase",
            }
          }
        >
           Michael Seiranian 
        </Text>
      </View>
      <View
        style={
          {
            "flexDirection": "row",
          }
        }
      >
        <View
          accessibilityState={
            {
              "busy": undefined,
              "checked": undefined,
              "disabled": undefined,
              "expanded": undefined,
              "selected": undefined,
            }
          }
          accessibilityValue={
            {
              "max": undefined,
              "min": undefined,
              "now": undefined,
              "text": undefined,
            }
          }
          accessible={true}
          collapsable={false}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            {
              "alignItems": "center",
              "borderRadius": 5,
              "display": "flex",
              "justifyContent": "center",
              "marginRight": "60%",
              "marginVertical": 15,
              "opacity": 1,
              "paddingHorizontal": 28,
              "paddingVertical": -10,
            }
          }
          testID="GithubButtonTestMatushan"
        >
          <Image
            source={1}
            style={
              {
                "aspectRatio": 1,
                "height": 50,
                "width": "100%",
              }
            }
          />
        </View>
        <Text
          style={
            {
              "alignSelf": "center",
              "color": "#fff",
              "fontSize": 18,
              "fontWeight": "500",
              "marginLeft": "-57%",
              "textTransform": "uppercase",
            }
          }
        >
           Matushan Yogaraj 
        </Text>
      </View>
      <View
        accessibilityState={
          {
            "busy": undefined,
            "checked": undefined,
            "disabled": undefined,
            "expanded": undefined,
            "selected": undefined,
          }
        }
        accessibilityValue={
          {
            "max": undefined,
            "min": undefined,
            "now": undefined,
            "text": undefined,
          }
        }
        accessible={true}
        collapsable={false}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          {
            "opacity": 1,
          }
        }
      >
        <Text
          style={
            {
              "alignSelf": "center",
              "backgroundColor": "purple",
              "borderRadius": 25,
              "color": "black",
              "fontSize": 20,
              "fontWeight": "bold",
              "padding": "2%",
              "textAlign": "center",
              "width": "60%",
            }
          }
        >
          Back to About Us
        </Text>
      </View>
    </View>
  </RCTScrollView>
</View>
`;
